domain/credential/service/service_test.go

```diff
@@ -275,6 +275,30 @@ func (s *serviceSuite) TestCheckAndUpdateCredential(c *tc.C) {
 	}})
 }
 
+// TestCheckAndUpdateNewCredential checks that a new credential can
+// be created if one doesn't exit already.
+func (s *serviceSuite) TestCheckAndUpdateNewCredential(c *tc.C) {
+	defer s.setupMocks(c).Finish()
+
+	cred := cloud.Credential{}
+	key := corecredential.Key{
+		Cloud: "cirrus",
+		Owner: usertesting.GenNewName(c, "bob"),
+		Name:  "foobar",
+	}
+
+	s.state.EXPECT().ModelsUsingCloudCredential(gomock.Any(), key).Return(nil, credentialerrors.NotFound)
+
+	s.state.EXPECT().UpsertCloudCredential(gomock.Any(), key, credential.CloudCredentialInfo{})
+
+	service := s.service(c)
+
+	results, err := service.CheckAndUpdateCredential(c.Context(), key,
```

Comment:
Can you give some order formatting to this line? Maybe just bring all the arguments down to the new line.